src/test/javascript/spec/component/text-exercise/tutor-effort-statistics.component.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { ComponentFixture, TestBed, getTestBed } from '@angular/core/testing';
+import { MockSyncStorage } from '../../helpers/mocks/service/mock-sync-storage.service';
+import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
+import { MockPipe, MockDirective } from 'ng-mocks';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { MockTranslateService } from '../../helpers/mocks/service/mock-translate.service';
+import { TranslateService } from '@ngx-translate/core';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { TutorEffortStatisticsComponent } from 'app/exercises/text/manage/tutor-effort/tutor-effort-statistics.component';
+import { ArtemisTestModule } from '../../test.module';
+import { MockHasAnyAuthorityDirective } from '../../helpers/mocks/directive/mock-has-any-authority.directive';
+import { ChartsModule } from 'ng2-charts';
+
+describe('TutorEffortStatisticsComponent', () => {
+    let fixture: ComponentFixture<TutorEffortStatisticsComponent>;
+    let component: TutorEffortStatisticsComponent;
+    let httpMock: HttpTestingController;
+    let compiled: any;
+    const tutorEffortsMocked = [
+        {
+            courseId: 1,
+            exerciseId: 1,
+            numberOfSubmissionsAssessed: 1,
+            totalTimeSpentMinutes: 1,
+        },
+        {
+            courseId: 1,
+            exerciseId: 1,
+            numberOfSubmissionsAssessed: 1,
+            totalTimeSpentMinutes: 20,
+        },
+        {
+            courseId: 1,
+            exerciseId: 1,
+            numberOfSubmissionsAssessed: 1,
+            totalTimeSpentMinutes: 30,
+        },
+    ];
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ChartsModule, ArtemisTestModule, HttpClientTestingModule],
+            declarations: [TutorEffortStatisticsComponent, MockPipe(ArtemisTranslatePipe), MockDirective(MockHasAnyAuthorityDirective)],
+            providers: [
+                { provide: LocalStorageService, useClass: MockSyncStorage },
+                { provide: SessionStorageService, useClass: MockSyncStorage },
+                { provide: TranslateService, useClass: MockTranslateService },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(TutorEffortStatisticsComponent);
+                component = fixture.componentInstance;
+                compiled = fixture.debugElement.nativeElement;
+                fixture.detectChanges();
+            });
+        httpMock = getTestBed().get(HttpTestingController);
+    });
+
+    it('should initialize', () => {
+        fixture.detectChanges();
+        expect(component).toBeTruthy();
```

Comment:
since you introduced this suite newly, you should use sinon chai for assertions as described in the [docs](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-tests/)

Reply:
done